features/step_definitions/navigation.rb

```diff
@@ -1,6 +1,6 @@
+# Open the options menu and select the given entry
 Then (/^I select "([^\"]*)" menu item$/) do |entry|
-  press_menu_button()
-  touch("* {text CONTAINS[c] '#{entry}'}")
+  select_options_menu_item(entry)
 end
```

Comment:
Hmm, looks like there is a built in command for this: https://github.com/calabash/calabash-android/blob/master/ruby-gem/lib/calabash-android/steps/navigation_steps.rb#L23